src/components/media_manager/src/media_manager_impl.cc

```diff
@@ -96,13 +96,15 @@ void MediaManagerImpl::set_mock_mic_recorder(MediaAdapterImpl* media_adapter) {
 
 #endif  // EXTENDED_MEDIA_MODE
```

Comment:
@istoilovgithub, extended_media_mode is turned off by default and currently build isn't broken.
Besides, I had this macro in my test, if we need use set_mock_mic_recorder.